py/test/selenium/selenium_test_suite.py

```diff
@@ -29,6 +30,7 @@ def suite():
         unittest.makeSuite(test_default_server.TestDefaultServer),
         unittest.makeSuite(test_google.TestGoogle),
         unittest.makeSuite(test_i18n.TestI18n),
+        unittest.makeSuite(test_remote_connection.TestRemoteConnection),
```

Comment:
As @lukeis said, this isnt used, you need to create a remote directory in https://github.com/SeleniumHQ/selenium/tree/master/py/test/selenium/webdriver and add the test file there.